src/jit/gentree.cpp

```diff
@@ -12938,6 +12938,61 @@ bool GenTree::IsLocalAddrExpr(Compiler* comp, GenTreeLclVarCommon** pLclVarTree,
     return false;
 }
 
+//------------------------------------------------------------------------
+// IsLclVarUpdateTree: Determine whether this is an assignment tree of the
+//                     form Vn = Vn 'oper' 'otherTree' where Vn is a lclVar
```

Comment:
Is `Vn` a shorthand for variable? `var` or `V` would be better, `Vn` is too close to "value number" :)